modules/light-clients/08-wasm/types/codec.go

```diff
@@ -27,6 +27,14 @@ func RegisterInterfaces(registry codectypes.InterfaceRegistry) {
 		(*sdk.Msg)(nil),
 		&MsgStoreCode{},
 	)
+	registry.RegisterImplementations(
```

Comment:
sweet, remember to add test cases for the registraction in `codec_test`.

Reply:
done